pkg/ansible/watches/watches.go

```diff
@@ -136,10 +136,45 @@ func (w *Watch) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	w.WatchClusterScopedResources = tmp.WatchClusterScopedResources
 	w.Finalizer = tmp.Finalizer
 	w.AnsibleVerbosity = getAnsibleVerbosity(gvk, ansibleVerbosityDefault)
+	w.addRolePlaybookPaths()
 
 	return nil
 }
 
+// addRolePlaybookPaths will add the full path
+// based on the current dir
+func (w *Watch) addRolePlaybookPaths() {
+	w.Playbook = getFullPath(w.Playbook)
+	w.Role = getFullRolePath(w.Role)
+	if w.Finalizer != nil && len(w.Finalizer.Role) > 0 {
+		w.Finalizer.Role = getFullRolePath(w.Finalizer.Role)
+	}
+	if w.Finalizer != nil && len(w.Finalizer.Playbook) > 0 {
+		w.Finalizer.Playbook = getFullPath(w.Finalizer.Playbook)
+	}
+}
+
+// getFullPath will return a valid full path for the playbook
+func getFullPath(path string) string {
+	if len(path) > 0 && !filepath.IsAbs(path) {
+		return filepath.Join(projutil.MustGetwd(), path)
+	}
+	return path
+}
+
+// getFullRolePath will return a valid full path for the role
+func getFullRolePath(path string) string {
+	if len(path) > 0 && !filepath.IsAbs(path) {
+		if len(os.Getenv("ANSIBLE_ROLES_PATH")) > 0 {
+			return filepath.Join(os.Getenv("ANSIBLE_ROLES_PATH"), path)
```

Comment:
It looks like `ANSIBLE_ROLES_PATH` is a colon-separated path, so this line will fail when there are multiple paths in the environment variable. Also, do we need to account for the default value listed in the following link?

Is there a way we can just let ansible-runner take care of this for us when we call it, rather than trying to come up with the right absolute path ourselves?

See https://docs.ansible.com/ansible/latest/reference_appendices/config.html#default-roles-path